### PR TITLE
should fix the bug related to config override within a project's dev-…

### PR DESCRIPTION
**Pull request by phuonglm**

### CODE REVIEW

**Review by datphan — APPROVED** [review]
working without warning

File: lib/utility.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Utility functions
-def overrides(obj1, obj2)
+def overrides(obj1, obj2, debug = false)
```
Comment (hoatle) [inline lib/utility.rb:2]: I'm not sure about debug here, does it do anything?

**Comment by hoatle** [comment]
@phuonglm please add unit tests so that we can see the bugs and the tests should be passed with the changes.

**Review by hoatle — CHANGES_REQUESTED** [review]
changes required

**Comment by hoatle** [comment]
closed, v0.6.0-a1 will fix this.

**Pull request closed without merging**